WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -262,10 +263,9 @@ private void resetEditTextContent(EditText editText) {
         }
     }
 
-    private String loadDefaultRole() {
-        int roleRes = isPrivateSite() ? R.array.invite_roles_private : R.array.invite_roles_public;
-        final String[] roles = getResources().getStringArray(roleRes);
-        return roles[0];
+    private Role loadDefaultRole() {
```

Comment:
Agreed, changed in 394c91e8d04004f901115beb6f04e5bf989ca77e.

@hypest if there was a specific reason you opted for `loadDefaultRole` rather than `getDefaultRole` in the original implementation, let me know and I can revert the change.